src/control/jobs/control_jobs.c

```diff
@@ -1829,50 +1830,60 @@ void dt_control_export(GList *imgid_list, int max_width, int max_height, int for
   mstorage->export_dispatched(mstorage);
 }
 
-static int32_t dt_control_time_offset_job_run(dt_job_t *job)
+static int32_t dt_control_datetime_job_run(dt_job_t *job)
 {
   dt_control_image_enumerator_t *params = (dt_control_image_enumerator_t *)dt_control_job_get_params(job);
   uint32_t cntr = 0;
   double fraction = 0.0;
   GList *t = params->index;
-  const long int offset = ((dt_control_time_offset_t *)params->data)->offset;
+  const long int offset = ((dt_control_datetime_t *)params->data)->offset;
+  const char *datetime = ((dt_control_datetime_t *)params->data)->datetime;
   char message[512] = { 0 };
 
   /* do we have any selected images and is offset != 0 */
-  if(!t || offset == 0)
+  if(!t || (offset == 0 && !datetime[0]))
   {
     return 1;
   }
 
   const guint total = g_list_length(t);
 
-  snprintf(message, sizeof(message),
-           ngettext("adding time offset to %d image", "adding time offset to %d images", total), total);
+  const char *mes11 = offset ? "adding time offset to %d image" : "setting date time to %d image";
```

Comment:
oups you missed the `N_(" ... ")` for the string to be added to the catalog. I'll do that.

Reply:
> I'll do that.

Je n'ai pas été bon sur ce coup. Désolé. Merci pour la correction.